gyp_xwalk

```diff
@@ -354,6 +354,18 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  parameter = '-Dverify_xwalk_apk='
```

Comment:
I agree that this should not be related to Debug vs Release. As you said that "set this in build/common.gypi instead like a regular variable", do you mean the users have to modify the parameter in common.gypi if they want to enable APK verification?